packages/web/stores/derived-data/concentrated-liquidity/historical-and-liquidity-data.ts

```diff
@@ -45,6 +47,11 @@ export class ObservableHistoricalAndLiquidityData {
     protected readonly queryTokenPairHistoricalPrice: DeepReadonly<ObservableQueryTokensPairHistoricalChart>
   ) {
     makeObservable(this);
+
+    // Init last hover price to current price in pool once loaded
+    autorun(() => {
+      if (this.lastChartData) this.setHoverPrice(this.lastChartData.close);
```

Comment:
Is there a risk for this to override the current hover value when data is refreshed? It could be clunky when it happens.

Reply:
Yeah, but the when hovering the the chart's value is rapidly set.